ReactWindows/ReactNative.Net46/Views/Text/ReactTextShadowNode.cs

```diff
@@ -213,8 +213,8 @@ private static YogaSize MeasureText(ReactTextShadowNode textNode, YogaNode node,
                 var normalizedHeight = YogaConstants.IsUndefined(height) ? double.PositiveInfinity : height;
                 textBlock.Measure(new Size(normalizedWidth, normalizedHeight));
                 return MeasureOutput.Make(
-                    (float)textBlock.DesiredSize.Width,
-                    (float)textBlock.DesiredSize.Height);
+                    (float)Math.Ceiling(textBlock.DesiredSize.Width),
```

Comment:
does that mean we should publish a new package (same hash they published to npm/cocoapods), and then update our version reference to fix the issue?